variables.tf

```diff
@@ -3,19 +3,23 @@ variable "vault_addr" {
 }
 
 variable "bootstrap_version" {
-  default = "v2.2.0"
+  default = "v2.6.4"
```

Comment:
could be `v2.6.5` though

Reply:
https://github.com/aeternity/terraform-aws-mainnet-api/pull/3/commits/d9c2b65d96b1b5128e682f64a80dd9668c17b32b